<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MvbState</class>
 <widget class="QWidget" name="MvbState">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 18px, &quot;微软雅黑&quot;;
}

QWidget
{
	background-color: black;
}</string>
  </property>
  <widget class="QLabel" name="lblRldC2">
   <property name="geometry">
    <rect>
     <x>677</x>
     <y>230</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLatB2">
   <property name="geometry">
    <rect>
     <x>766</x>
     <y>192</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>192</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>RIOM LAT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>230</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>RIOM RLD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRldB2">
   <property name="geometry">
    <rect>
     <x>766</x>
     <y>230</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLatD1">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>192</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRldD2">
   <property name="geometry">
    <rect>
     <x>588</x>
     <y>230</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLatB1">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>192</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRldB1">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>230</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRldD1">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>230</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRldA2">
   <property name="geometry">
    <rect>
     <x>855</x>
     <y>230</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLatD2">
   <property name="geometry">
    <rect>
     <x>588</x>
     <y>192</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLatA1">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>192</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRldA1">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>230</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLatC1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>192</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLatA2">
   <property name="geometry">
    <rect>
     <x>855</x>
     <y>192</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRldC1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLatC2">
   <property name="geometry">
    <rect>
     <x>677</x>
     <y>192</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>927</x>
     <y>540</y>
     <width>85</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 18px, &quot;微软雅黑&quot;;
    color: black;
	background-color: rgb(113, 113, 198);
	border:2px solid white; 
    border-right-color: rgb(113, 113, 198); 
    border-bottom-color: rgb(113, 113, 198);
}

QPushButton:disabled
{
	font: 18px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(113, 113, 198);
	border:2px solid white; 
    border-right-color: rgb(113, 113, 198); 
    border-bottom-color: rgb(113, 113, 198);
}

QPushButton:pressed
{
	font: 18px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 0, 255); 
	border:2px solid white; 
	border-left-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, 
                        stop: 0 gray, stop: 1 black); 			border-top-color: qlineargradient(x1: 0, y1: 0, x2: 0, 										y2: 1, stop: 0 gray, stop: 1 black);
  border-right-color: white; border-bottom-color: white;
}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAirCondiitionerDeloadC1">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>292</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>330</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>空调减载(CCU)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblAirCondiitionerDeloadD1">
   <property name="geometry">
    <rect>
     <x>497</x>
     <y>292</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblAirCondiitionerDeloadA2">
   <property name="geometry">
    <rect>
     <x>853</x>
     <y>292</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblAirCondiitionerDeloadC2">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>292</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblAirCondiitionerDeloadB2">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>292</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblAirCondiitionerDeloadB1">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>292</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>292</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>空调减载(ACU)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblAirCondiitionerDeloadD2">
   <property name="geometry">
    <rect>
     <x>586</x>
     <y>292</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblAirCondiitionerDeloadA1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>292</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>392</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>CCU Time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCcuTime">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>392</y>
     <width>714</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCcuAirCondiitionerDeloadA1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCcuAirCondiitionerDeloadB1">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>330</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCcuAirCondiitionerDeloadC2">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>330</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCcuAirCondiitionerDeloadD1">
   <property name="geometry">
    <rect>
     <x>497</x>
     <y>330</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCcuAirCondiitionerDeloadB2">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>330</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCcuAirCondiitionerDeloadD2">
   <property name="geometry">
    <rect>
     <x>586</x>
     <y>330</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCcuAirCondiitionerDeloadA2">
   <property name="geometry">
    <rect>
     <x>853</x>
     <y>330</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCcuAirCondiitionerDeloadC1">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>330</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
